.circleci/config.yml

```diff
@@ -77,6 +77,11 @@ jobs:
       image_tag: << parameters.image_tag >>
     steps:
       - installation
+      - run:
+          name: "Verify installed packages have compatible dependencies."
+          command: |
+            check=$(pip check); echo $check
```

Comment:
I believe pip check will exit with code 1 if it finds an inconsistency.  We may not need to verify what is printed out, just have circle ci check the exit code.